server/models/requestModel.ts

```diff
@@ -14,6 +14,9 @@ interface RequestInterface {
 type RequestDocument = RequestInterface & Document;
 
 const requestSchema = new Schema({
+  requestType: {
```

Comment:
Could we get `requestType` added to `RequestInterface`?

Reply:
Added